NRLMMD-GEOIPS/geoips
# 264 update seviri reader to use pypublicdecompwt

# Reviewer Checklist
https://github.com/NRLMMD-GEOIPS/.github/blob/main/.github/review-template.md

# Related Issues
fixes NRLMMD-GEOIPS/geoips#264


# Testing Instructions
- Unset `XRIT_DECOMPRESS_PATH`
- Check out this branch.
- Reinstall GeoIPS via `pip install -e .`
- Attempt to read SEVIRI data. Maybe run a test script if available.

# Summary
Enhancements
============

Replace xRITDecompress with pyPublicDecompWT for seviri_hrit reader
-------------------------------------------------------------------

*From NRLMMD-GEOIPS/geoips#264: 2023-08-16, Update seviri reader to use pyPublicDecompWT*

* We had previously been using xRITDecompress which needed to be complied and installed
  separately. This replaces xRITDecompress with pyPublicDecompWT which provides the same
  functionality but can be pip installed.

::

    modified: geoips/plugins/modules/readers/utils/hrit_reader.py

Installation Updates
====================

Add pyPublicDecompWT to dependencies and remove setup_seviri from setup script
------------------------------------------------------------------------------

*From NRLMMD-GEOIPS/geoips#264: 2023-08-16, Update seviri reader to use pyPublicDecompWT*

* Add pypublicdecompwt to install requirements
* Remove setup_seviri from setup.py
* Remove xRITDecompress environment variables from config_geoips

::

    modified: pyproject.toml
    modified: setup_seviri
    modified: config/config_geoips

# Output
